src/translations/screens/subscreens/PurchaseOverview.ts

```diff
@@ -8,6 +8,11 @@ const PurchaseOverviewTexts = {
       'Oops - vi klarte ikke å søke opp prisen. Supert om du prøver igjen 🤞',
       'Whoops - we were unable to retrieve cost. Please try again 🤞',
     ),
+    producUnavailable: (productName: string) =>
+      _(
+        `Produktet ${productName} er kanskje ikke tilgjengelig akkurat nå.`,
+        `Product ${productName} might not be available right now.`,
+      ),
```

Comment:
@ckbilstad What do you think about these texts? ;)

Reply:
I had a talk with Christian and the texts were changed as the image below:

![Screenshot_1673947628](https://user-images.githubusercontent.com/43450882/212861216-65adf0a5-50ee-4c3e-9792-8859e6abf4b4.png)